src/Lykke.AlgoStore.Api/RealTimeStreaming/DataStreamers/WebSockets/Handlers/TradesWebSocketHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Common.Log;
+using Lykke.AlgoStore.Algo.Charting;
+using Lykke.AlgoStore.Api.RealTimeStreaming.Filters;
+using Lykke.AlgoStore.Api.RealTimeStreaming.Sources;
+using Microsoft.AspNetCore.Http;
+
+namespace Lykke.AlgoStore.Api.RealTimeStreaming.DataStreamers.WebSockets.Handlers
+{
+    public class TradesWebSocketHandler : WebSocketHandlerBase<TradeChartingUpdate>
```

Comment:
Can these classes be made more generic? The constructors are more or less the same

Reply:
Please see the overall comment below.